.github/workflows/branch-tests.yml

```diff
@@ -15,7 +15,8 @@ jobs:
 
     strategy:
       matrix:
-        node-version: 18
+        node-version: [ 18 ]
+        os: [ubuntu-latest]
```

Comment:
what's the reason for this change? 
I'm not against it to be honest but perhaps it's worth having a different issue and PR for such a change?

Reply:
It can be removed. The pipelines were breaking on my fork and this was reported as a syntax error. 🚧

Reply:
removed.

Reply:
Thanks for removing it, I'll check why it gave you that error and in case I'll open an issue ;) 
thank you @jaalru :)